src/main/java/com/cinema/controllers/CinemaHallController.java

```diff
@@ -0,0 +1,42 @@
+package com.cinema.controllers;
+
+import com.cinema.model.CinemaHall;
+import com.cinema.model.dto.CinemaHallRequestDto;
+import com.cinema.model.dto.CinemaHallResponseDto;
+import com.cinema.service.CinemaHallService;
+import com.cinema.service.mapper.CinemaHallMapper;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema-halls")
+public class CinemaHallController {
+    private final CinemaHallService cinemaHallService;
+    private final CinemaHallMapper cinemaHallMapper;
+
+    public CinemaHallController(CinemaHallService cinemaHallService,
+                                CinemaHallMapper cinemaHallMapper) {
+        this.cinemaHallService = cinemaHallService;
+        this.cinemaHallMapper = cinemaHallMapper;
+    }
+
+    @PostMapping
+    public CinemaHall addCinemaHall(@RequestBody CinemaHallRequestDto cinemaHallRequestDto) {
```

Comment:
let's not return the model itself (check all controllers)

return either void or ResponseDto